Openwebif has orgdream check for dm8000 in model branding.
Which result in no full remote control in openwebif and no brand 'dream multimedia' in info screen.
Every dm8000 is considered as a clone box in openpli, even your dm8000 is genuine.

diff --git a/plugin/controllers/models/owibranding.py b/plugin/controllers/models/owibranding.py
index 391ce28c..25bcef4e 100644
--- a/plugin/controllers/models/owibranding.py
+++ b/plugin/controllers/models/owibranding.py
@@ -356,13 +356,9 @@ def getAllInfo():
 				model = "Spark 7162"
 			else:
 				model = "Spark"
-		elif (procmodel.startswith("dm") and not procmodel == "dm8000"):
+		elif (procmodel.startswith("dm")):
 			brand = "Dream Multimedia"
 			model = procmodel.replace("dm", "DM", 1)
-		# A "dm8000" is only a Dreambox if it passes the tpm verification:
-		elif procmodel == "dm8000" and orgdream:
-			brand = "Dream Multimedia"
-			model = "DM8000"
 		else:
 			model = procmodel
 
@@ -473,7 +469,7 @@ def getAllInfo():
 		remote = procmodel.replace(" ", "")
 	elif procmodel == "vg2000":
 		remote = "xcombo"
-	elif procmodel == "dm8000" and orgdream:
+	elif procmodel == "dm8000":
 		remote = "dmm1"
	elif procmodel in ("dm7080", "dm7020hd", "dm7020hdv2", "dm800sev2", "dm500hdv2", "dm520", "dm820", "dm900"):
 		remote = "dmm2"
